drivers/counter/timer_tmr_cmsdk_apb.c

```diff
@@ -87,9 +87,10 @@ static u32_t timer_tmr_cmsdk_apb_read(struct device *dev)
 	return value;
 }
 
-static int timer_tmr_cmsdk_apb_set_alarm(struct device *dev,
-					 counter_callback_t callback,
-					 u32_t count, void *user_data)
+static int timer_tmr_cmsdk_apb_set_wrap_alarm(struct device *dev,
+					      u32_t ticks,
+					      counter_wrap_callback_t callback,
+					      void *user_data)
```

Comment:
same here. `if (data->status == TIMER_DISABLED) {` can be removed.

Reply:
Agreed. I will address this in v3